src/store-api/src/logstore/entry.rs

```diff
@@ -143,6 +144,15 @@ impl Entry {
         }
     }
 
+    pub fn into_buffer(self) -> Buffer {
```

Comment:
I prefer moving function into `mito2` crate.

Reply:
But the Entry struct in the object-store crate. Wouldn’t it be more intuitive to implement it there?

Reply:
But the `Entry` struct is in the `store-api` crate. Wouldn’t it be more intuitive to implement it there?

Reply:
I think it's better to keep `store-api` crate simple.t

Reply:
ok I will write it in `src/mito2/src/wal/entry_reader.rs` , by the way how to fix  Toml check ,I aready run `make fmt-toml`,but it still error in CI. I also run `taplo format check ` on my pc,but no Error.
```shell
taplo format --check
 INFO taplo:format_files:load_config: found configuration file path="/home/wr/workspace/greptimedb/taplo.toml"
 INFO taplo:format_files:collect_files: found files total=61 excluded=0 cwd="/home/wr/workspace/greptimedb"
```